pyfolio/plotting.py

```diff
@@ -1726,17 +1688,26 @@ def plot_multistrike_cones(is_returns, oos_returns, num_samples=1000,
         Plot title
     ax : matplotlib.Axes, optional
         Axes upon which to plot.
-    cone_std : int, float, or list of int/float
+    cone_std : list of int/float
         Number of standard devations to use in the boundaries of
         the cone. If multiple values are passed, cone bounds will
         be generated for each value.
     random_seed : int
         Seed for the pseudorandom number generator used by the pandas
         sample method.
+    num_strikes : int
+        Upper limit for number of cones drawn. Can be anything from 0 to 3.
```

Comment:
Minor nit on the docstring here - if num_strikes is 0, then we'll still have one cone drawn(even though we say in the docstring that the upper limit is 0). What do you think of rephrasing to something like:
```
num_strikes : int
  Number of times to re-draw a cone if we fall outside the bounds of the previous one. Can be any int from 0 to 3.
```